classes/class-twentytwenty-svg-icons.php

```diff
@@ -10,7 +10,7 @@
 if ( ! class_exists( 'TwentyTwenty_SVG_Icons' ) ) {
 	/**
 	 * SVG ICONS CLASS
-	 * Retrieve the SVG code for the specified icon. Based on a solution in TwentyNineteen.
+	 * Retrieve the SVG code for the specified icon. Based on a solution in Twenty Nineteen.
```

Comment:
This one should probably say Twenty Twenty

Reply:
I thought so too initially, but it says _"Based on a solution in Twenty Nineteen."_ Thus, it refers to the source of that code snippet. There's one more similar comment in `classes/class-twentytwenty-walker-comment.php:13`:

> A custom walker for comments, based on the walker in Twenty Nineteen.

As both parts referring to the source, I would not change them.

Reply:
I should've read the whole thing ;) thanks for the clarification.